modules/reference/pages/rpk/rpk-cluster/rpk-cluster-partitions-list.adoc

```diff
@@ -0,0 +1,86 @@
+= rpk cluster partitions list
+:description: rpk cluster partitions list
+
+List partitions in the cluster.
+
+This commands lists the cluster-level metadata of all partitions in the cluster.
+
+It shows the current replica assignments on both brokers and CPU cores for given topics. By default, it assumes the `kafka` namespace, but you can specify an internal namespace using the `{namespace}/` prefix.
+
+The `REPLICA-CORE` column displayed in the output table contains a list of replicas assignments in the form of: <Node-ID>-<Core>.
+
+If the DISABLED column contains a '-' value, then it means you are running this command against a cluster that does not support the underlying API.
+
+== Enabled/Disabled
+
+
+Disabling a partition in Redpanda involves prohibiting any data consumption or production to and from it. All internal processes associated with the partition are stopped, and it remains unloaded during system startup. This measure aims to maintain cluster health by preventing issues caused by specific corrupted partitions that may lead to Redpanda crashes. Although the data remains stored on disk, Redpanda ceases interaction with the disabled partitions to ensure system stability. You may disable/enable partition using `rpk cluster partitions enable/disable`.	
+
+
+
+== Examples
+
+List all partitions in the cluster:
+
+```bash
+rpk cluster partitions list --all
+```
+
+List all partitions in the cluster, filtering for topic foo and bar:
+
+```bash
+rpk cluster partitions list foo bar
+```
+
+List only the disabled partitions:
+
+```bash
+rpk cluster partitions list -a --disabled-only
+```
+
+
+
+List all in json format:
+
+```bash
+rpk cluster partition list -a --format json
+```
+
+== Usage
+
+[,bash]
+----
+rpk cluster partitions list [TOPICS...] [flags]
+----
+
+== Aliases
+
+[,bash]
+----
+list, ls, describe
+----
+
+== Flags
+
+[cols="1m,1a,2a"]
+|===
+|*Value* |*Type* |*Description*
+
+|-a, --all |- |If true, list all partitions in the cluster.
```

Comment:
Having the `m` on the column will make this look like `-a, --all`. We should instead wrap each option in separate backticks to avoid the comma getting included.

Reply:
Like discussed on slack will be using this format in the future

```suggestion
a|`-a` or `--all` |- |If true, list all partitions in the cluster.
```